adhocracy4/comments_async/static/comments_async/comment.jsx

```diff
@@ -254,8 +254,12 @@ export default class Comment extends React.Component {
 
   render () {
     let lastDate
-    if (this.props.modified === null || this.props.is_deleted) {
+    if (this.props.modified === null) {
       lastDate = localeDate(this.props.created)
+    } if (this.props.is_deleted && this.isOwner()) {
```

Comment:
The `this.isOwner()` is about who wrote the comment, I think. If you want to distinguish between the user deleting it and the moderator you should use `is_deleted` for the user and `is_censored` for the moderator.

Reply:
Perfect! Thanks, will try this